### PR TITLE
infowindows: type in tabmenu

### PR DESCRIPTION
**Pull request by matallo**
close https://github.com/CartoDB/cartodb/issues/7804

screen shot 2016-06-20 at 18 19 01

CR @xavijam /cc @saleiva

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 var CoreView = require('backbone/core-view');
+var Template = require('./tab-pane-label.tpl');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js]: Template is not a class, better to use it in lowercase please.

File: lib/assets/javascripts/cartodb3/components/tab-pane/create-text-labels-tab-pane.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ module.exports = function (paneItems, options) {
       name: paneItem.name,
       selected: paneItem.selected,
       label: paneItem.label,
+      selectedChild: paneItem.selectedChild,
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/tab-pane/create-text-labels-tab-pane.js]: What is that?

**Comment by xavijam** [comment]
Please, let's not add the templates we used to have in order to solved other problem. We will take care of it in other issue, so please, let's not add them.

File: lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js (hunk 1 above)
Comment (matallo) [inline lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js]: found it in many places, shall I replace all of them? :/

i.e. https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-icon-view.js#L2

File: lib/assets/javascripts/cartodb3/components/tab-pane/create-text-labels-tab-pane.js (hunk 2 above)
Comment (matallo) [inline lib/assets/javascripts/cartodb3/components/tab-pane/create-text-labels-tab-pane.js]: It's the "none" in the "click none", and the "light" in the "hover light", it's what let us have the selected child in the tabs

File: lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js (hunk 1 above)
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js]: Nah, just replace yours.

File: lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js [hunk 3]
```diff
@@ -14,7 +15,7 @@ module.exports = CoreView.extend({
   },
 
   render: function () {
-    this.$el.html(this.model.get('label'));
+    this.$el.html(Template(this.model.toJSON()));
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js]: ^^^

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js]: Please specify what we need, label and selectedChild, if it is not defined, just set as '', for example.

File: lib/assets/locale/en.json [hunk 4]
```diff
@@ -605,10 +605,21 @@
           "window-size": "Window size",
           "header-color": "Header color",
           "none": "None",
-          "light": "Light",
-          "dark": "Dark",
-          "color": "Color",
-          "image": "Image"
+          "infowindow_light": "Light",
```
Comment (xavijam) [inline lib/assets/locale/en.json]: Could we use hyphens instead of underscores?

Comment (matallo) [inline lib/assets/locale/en.json]: I'm sorry not, they have to match the actual template names :S

**Comment by matallo** [comment]
feedback applied

**Comment by xavijam** [comment]
:+1:

File: lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js [hunk 5]
```diff
@@ -14,7 +15,10 @@ module.exports = CoreView.extend({
   },
 
   render: function () {
-    this.$el.html(this.model.get('label'));
+    this.$el.html(template({
+      label: this.model.get('label'),
+      selectedChild: this.model.get('selectedChild') ? this.model.get('selectedChild') : ''
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/tab-pane/tab-pane-label-view.js]: this is the same: 
```js
this.model.get('selectedChild') || ''
```

**Pull request merged**